public/app/features/plugins/PluginsErrorsInfo.tsx

```diff
@@ -47,16 +47,8 @@ export const PluginsErrorsInfoUnconnected: React.FC<PluginsErrorsInfoProps> = ({
     >
       <div>
         <p>
-          We have encountered{' '}
-          <a
-            href="https://grafana.com/docs/grafana/latest/developers/plugins/backend/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            data source backend plugins
-          </a>{' '}
-          that are unsigned. Grafana Labs cannot guarantee the integrity of unsigned plugins and recommends using signed
-          plugins only.
+          We have encountered plugins that are unsigned. Grafana Labs cannot guarantee the integrity of unsigned plugins
```

Comment:
"encountered" seems odd here. Wouldn't it be something about "You attempted to load unsigned plugins" or "You have opened Grafana with unsigned plugins installed.""?

For the second sentence, I would reverse the order.

Grafana Labs cannot guarantee the integrity of unsigned plugins. We recommend only using signed plugins.

Reply:
It's a little more nuanced I suppose. So Grafana will try to load all plugins within the relevant plugin folder - and users can put whatever they want to in there. This warning will show them the list of plugins that Grafana found to be unsigned (not having a signature at all or having a bad signature). That's why it's more like _"hey Grafana found these things.."_